TestAssets/AsIsProjects/XprojCsprojProjects/BasicProject_invalid-missing-dll/Sample1-Xproj+Csproj/artifacts/src/ConsoleApp13/obj/Debug/net46/dotnet-compile.rsp

```diff
@@ -0,0 +1,20 @@
+--temp-output:C:\Users\micodoba\OneDrive for Business\tests\TestXprojCsproj-master\Sample1-Xproj+Csproj\artifacts\src\ConsoleApp13\obj\Debug\net46\
```

Comment:
The tooling end to end scenario is hard. We should clean up these info. We need the lock file and binary files since they are not created by `dotnet-cli`. They're VS artifact. but we need them for E2E test. We'll look for a better scenario.